### PR TITLE
Cleaned up demo sections

### PR DESCRIPTION
**Pull request by jmoreira-valory**
## Proposed changes

Simplified and cleaned up demo sections.
Changed images to the new color scheme.
Removed redundant information and redirected to corresponding deployment section.

There might be many files changed due to update in links.

## Fixes

If it fixes a bug or resolves a feature request, be sure to link to that issue.

## Types of changes

What types of changes does your code introduce? (A **breaking change** is a fix or feature that would cause existing functionality and APIs to not work as expected.)
_Put an `x` in the box that applies_

- [ ] Non-breaking fix (non-breaking change which fixes an issue)
- [ ] Breaking fix (breaking change which fixes an issue)
- [ ] Non-breaking feature (non-breaking change which adds functionality)
- [ ] Breaking feature (breaking change which adds functionality)
- [ ] Refactor (non-breaking change which changes implementation)
- [ ] Messy (mixture of the above - requires explanation!)

## Checklist

_Put an `x` in the boxes that apply._

- [ ] I have read the [CONTRIBUTING](../blob/main/CONTRIBUTING.md) doc
- [ ] I am making a pull request against the `main` branch (left side). Also you should start your branch off our `main`.
- [ ] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have locally run services that could be impacted and they do not present failures derived from my changes

## Further comments

If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: docs/demos/price_oracle_intro.md [hunk 1]
```diff
@@ -28,86 +27,41 @@ This demo is composed of:
...
-you have followed the [setup instructions](guides/quick_start.md#setup). As a result you should have a Pipenv workspace folder.
-
-2. Use the CLI to download the `valory/oracle_hardhat` service.
+1. Open a terminal and run a local [HardHat](https://hardhat.org/) node that will emulate a blockchain node. For convenience, we provide a Docker image in [Docker Hub](https://hub.docker.com/) that can be run by executing:
     ```bash
-    autonomy fetch valory/oracle_hardhat:0.1.0:bafybeie553shfmnds6v7defynjv5kmjkqf2aygj345jbbcssevtnkbodbe --remote --service
-    cd oracle_hardhat
-    ```
-
-3. Prepare a JSON file `keys.json` containing the addresses and keys of the four agents that are
```
Comment (angrybayblade) [inline docs/demos/price_oracle_intro.md:48]: Where is this now?

Comment (jmoreira-valory) [reply docs/demos/price_oracle_intro.md:48]: As you can see, the instructions guides have been simplified (this is less error-prone), and now the guides reference "macro-steps":. This is now subsumed in the "deploy local service guide", and is referenced in line 54 (right side):

```3. Follow the steps in the [local deployment guide](../guides/deploy_service.md#local-deployment) to deploy and run the service locally.```

**Review by DavidMinarsch — APPROVED** [review]

**Review by DavidMinarsch — CHANGES_REQUESTED** [review]
Please fix failing doc tests

**Comment by jmoreira-valory** [comment]
Linters seems working now.

**Review by dvilelaf — APPROVED** [review]
LGTM

**Review by angrybayblade — APPROVED** [review]

**Review by DavidMinarsch — APPROVED** [review]

**Pull request merged**